convert.py

```diff
@@ -1322,26 +1357,33 @@ def _create_special_vocab(self, vocab: Vocab, vocabtype: str, model_parent_path:
             n_vocab=n_vocab,
         )
 
-    def load_vocab(self, vocabtype: str, model_parent_path: Path) -> tuple[Vocab, gguf.SpecialVocab]:
+    def _create_vocab_by_path(self, vocabtype: str, model_parent_path: Path) -> Vocab:
         path = self._select_file(vocabtype)
         print(f"Loading vocab file '{path}', type '{vocabtype}'")
 
         added_tokens_path = path.parent / "added_tokens.json"
-        vocab: Vocab
         if vocabtype == "bpe":
-            vocab = BpeVocab(
+            return BpeVocab(
                 path, added_tokens_path if added_tokens_path.exists() else None
             )
         elif vocabtype == "spm":
-            vocab = SentencePieceVocab(
+            return SentencePieceVocab(
                 path, added_tokens_path if added_tokens_path.exists() else None
             )
         elif vocabtype == "hfft":
-            vocab = HfVocab(
+            return HfVocab(
                 path, added_tokens_path if added_tokens_path.exists() else None
             )
         else:
             raise ValueError(f"Unsupported vocabulary type {vocabtype}")
```

Comment:
Change the `elif`s here to `if`s, and remove the `else` from the `raise`.

Reply:
fixed